### PR TITLE
Hw 8

### PR DESCRIPTION
**Pull request by matthewlaz**
add notify, create resolutions, list of resolutions

### CODE REVIEW

File: src/components/doctor/patients/DropDownList.tsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-  onCreateResolution: () => void
+  onCreateResolution: () => void,
+  onEditResolution: () => void,
+  onDelete: (id: string) => void,
+  id: string,
 };
-
-const DropDownList = ({ onCreateResolution }: DropDownProps) => (
+// const { ref, isComponentVisible } = useComponentVisible(true);
+// console.log(isComponentVisible);
```
Comment (codezavr) [inline src/components/doctor/patients/DropDownList.tsx]: ???

File: src/components/doctor/patients/DropDownList.tsx [hunk 2]
```diff
@@ -1,17 +1,23 @@
 import { StyledItem, StyledList, StyledWrapper } from './DropDownList.styled';
+// import useComponentVisible from '../../../hooks/useComponentVisible';
```
Comment (codezavr) [inline src/components/doctor/patients/DropDownList.tsx]: ???

File: src/components/doctor/patients/modals/CreateResolution.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    };
+    createResolutionHandler(resolutionData);
+  };
+  return (
+    <Modal onClose={onClose}>
+      <StyledWrapper>
+        <StyledTitle>Create a Resolution</StyledTitle>
+        <StyledName>
+          {firstName}
+          {' '}
```
Comment (codezavr) [inline src/components/doctor/patients/modals/CreateResolution.tsx:34]: what is this for? Can we avoid it?

File: src/components/forms/SignInForm.tsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ import { StyledInputs, StyledLink } from './SignInForm.styled';
 import { StyledForm, StyledSidebar } from './LoginPages.styled';
 import Button from './button/Button';
 import { LogInType } from '../models/logIn.model';
+// import { Toast } from '../UI/Notify';
```
Comment (codezavr) [inline src/components/forms/SignInForm.tsx]: ??????????

File: src/hooks/useComponentVisible.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect, useRef } from 'react';
+
+export default function useComponentVisible(initialIsVisible: any) {
+  const [isComponentVisible, setIsComponentVisible] = useState(initialIsVisible);
+  const ref = useRef(null);
+
+  const handleClickOutside = (event: any) => {
```
Comment (codezavr) [inline src/hooks/useComponentVisible.ts:7]: Google event type

**Review by codezavr — CHANGES_REQUESTED** [review]

File: src/components/doctor/patients/DropDownList.tsx (hunk 1 above)
Comment (matthewlaz) [reply src/components/doctor/patients/DropDownList.tsx]: removed

File: src/components/doctor/patients/DropDownList.tsx (hunk 2 above)
Comment (matthewlaz) [reply src/components/doctor/patients/DropDownList.tsx]: removed

File: src/components/doctor/patients/modals/CreateResolution.tsx (hunk 3 above)
Comment (matthewlaz) [reply src/components/doctor/patients/modals/CreateResolution.tsx:34]: this is for space firstname and lastname. Airbnb rules don't allow me to have 1 more expression per line. If it is not ok i could create an exception. What do you think?

File: src/components/forms/SignInForm.tsx (hunk 4 above)
Comment (matthewlaz) [reply src/components/forms/SignInForm.tsx]: removed

File: src/hooks/useComponentVisible.ts (hunk 5 above)
Comment (matthewlaz) [reply src/hooks/useComponentVisible.ts:7]: removed. This file have already deleted )

File: src/components/doctor/patients/modals/CreateResolution.tsx (hunk 3 above)
Comment (codezavr) [reply src/components/doctor/patients/modals/CreateResolution.tsx:34]: Okay, let it be.

**Review by codezavr — APPROVED** [review]

**Pull request merged**